tests/testthat/test-verb-select.R

```diff
@@ -66,11 +66,16 @@ test_that("can rename with a function", {
 
 test_that("select preserves grouping vars", {
   mf <- memdb_frame(a = 1, b = 2) %>% group_by(b)
-  out <- mf %>% select(a) %>% collect()
+  expect_message(out <- mf %>% select(a) %>% collect())
```

Comment:
Could you add a snapshot test (or possibly modify this test) to ensure that we have eyes on the generate message?